### PR TITLE
Enable all available layers

### LINKED ISSUE #27: How to use other layers?
How to other layers like F.Adhes, Eco1.User etc?
I tried to add these layers with same name but they are not exported.

### PR DESCRIPTION
**Pull request by fdb-git**
Fixes #27

### CODE REVIEW

**Comment by kasbah** [comment]
Thanks! Can't test right now. Have you tested this, does it work as is?

**Comment by fdb-git** [comment]
I've fixed an issue with the first commit.
You can try with the test file added in my branch.

File: inkscape/svg2shenzhen/export.py [hunk 1]
```diff
@@ -410,8 +410,14 @@ def processExportLayer(self):
                     #export layer png to kicad
                     p = self.exportToKicad(layer_dest_png_path, layer_dest_kicad_path, layer_label, invert)
                     processes.append(p)
-                for p in processes:
```
Comment (kasbah) [inline inkscape/svg2shenzhen/export.py:400]: Why is this being removed?

**Comment by kasbah** [comment]
Please don't add test files to the inkscape folder. I don't think it makes sense adding this file to the code base at all unless it goes in an example folder with proper explanations or as part of automated tests (we should make some really). Please remove the last commit.

**Comment by urish** [comment]
@kasbah I'd love to see this PR merged. If you are interested, I can fork it, make the requested changes and submit a new PR. please let me know

**Comment by kasbah** [comment]
Sure! :wink:

**Comment by urish** [comment]
This can be closed now that #50 has been merged

**Pull request closed without merging**